src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo__link {
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding-right: 40px ;
```

Comment:
Tests failed. The reason can be here. Remove horizontal paddings for nav__list.
Instead set horizontal paddings for header.